src/main/java/org/folio/circulation/domain/RequestRepresentation.java

```diff
@@ -68,7 +68,9 @@ private static void addAdditionalItemProperties(JsonObject request, Item item) {
 
     if (location != null) {
       itemSummary.put("location", new JsonObject()
-        .put("name", location.getName()));
+        .put("name", location.getName())
```

Comment:
Personally, I would use the `JSONPropertyWriter` to protect this code from null values for these properties.

I'm aware this change has only followed the pattern in the previous change.

Reply:
Done

Reply:
This code does not appear to have changed, is that because the change hasn't been pushed yet?

Reply:
The changes have been pushed.